Tests/Acceptance/DoctrineAccessTokenManagerTest.php

```diff
@@ -109,9 +160,13 @@ public function testClearExpiredWithRefreshToken(): void
             timecop_return();
         }
 
+        $mapFunction = static function (RefreshToken $refreshToken): string {
+            return $refreshToken->getIdentifier();
+        };
+
         $this->assertSame(
-            $testData['output'],
-            $em->getRepository(RefreshToken::class)->findBy(['accessToken' => null], ['identifier' => 'ASC'])
+            array_map($mapFunction, $testData['output']),
+            array_map($mapFunction, $em->getRepository(RefreshToken::class)->findBy(['accessToken' => null], ['identifier' => 'ASC']))
```

Comment:
@franjo-zadelj-trikoder Why is this change needed?

Reply:
@HypeMC it's not anymore, I think in one iteration it was. Gonna revert.